analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/MetricsAnalyzerTest.cs

```diff
@@ -71,13 +71,13 @@ public void VerifyMetrics_Razor()
 
                     // ToDo: other metrics will be fixed in subsequent PRs.
                     metrics.ClassCount.Should().Be(1);
```

Comment:
ClassCount remains equal to 1, we added `public class MyClass { }` but we no longer count the razor-generated class